photon-lib/py/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+import subprocess
+
+versionStr = subprocess.check_output(['git', 'describe', '--tags', "--match=v*", "--always"]).decode('utf-8').strip()
```

Comment:
Consider using a build-system for this, e.g. `setuptools_scm`, which can then set your `version` appropriately.

Reply:
The current photonvision versioning scheme does not align with pep440, and it does not look like setuptools_scm has a hook early enough for me to intercept and transform the version?